runtime/moonbase/src/lib.rs

```diff
@@ -77,14 +77,16 @@ pub mod currency {
 	use super::Balance;
 
 	pub const UNITS: Balance = 1_000_000_000_000_000_000;
-	pub const CENTS: Balance = UNITS / 100;
-	pub const GRAND: Balance = UNITS * 1_000;
-	pub const MILLICENTS: Balance = CENTS / 1_000;
+	pub const MILLIUNITS: Balance = UNITS / 1000;
+	pub const MICROUNITS: Balance = MILLIUNITS / 1000;
+	pub const NANOUNITS: Balance = MICROUNITS / 1000;
 
-	pub const BYTE_FEE: Balance = 1 * MILLICENTS;
+	pub const KILOUNITS: Balance = UNITS * 1_000;
```

Comment:
Could we name them `MILLIS`, `MICROS`, etc so we can copy paste more easily between runtimes?

Or if you're feeling ambitious, we could move this whole `currency` module to a common place and use `UNITS` everywhere. That's a nice general name.

Reply:
But I'm also fine with this as it is.

Reply:
Good idea

Reply:
The team later decided that we'd prefer to keep the runtimes fully separate because:

1. They are for different live networks and will evolve independently
2. The confidence that changing one runtime does not change the others.